src/transformStateWithClones.js

```diff
@@ -8,6 +8,37 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+
+  const arrResult = [];
+  const cloneState = {
+    ...state,
+  };
+
+  for (const key of transforms) {
```

Comment:
якщо transforms це багато - то один  це буде transform
```suggestion
  for (const transform of transforms) {
```